data/core.yaml

```diff
@@ -732,6 +732,21 @@ en:
     add_fields: "Add field:"
     lock:
       suggestion: 'The "{label}" field is locked because there is a Wikidata tag. You can delete it or edit the tags in the "Tags" section.'
+    display_name:
+      direction: "{direction}"
```

Comment:
Do you think any languages need this and `ref` to be translatable? Seems unnecessary even if it makes the code cleaner.

Reply:
I wasn’t sure. Some languages like Chinese and Japanese tend to append a suffix to numbers or direction words, so if they feel the need to do that for the longer strings, they might want to do the same thing in these shorter strings for consistency. But I would expect most languages to leave these strings alone.

Reply:
Okay, I'll leave it alone then.